sesman/startwm.sh

```diff
@@ -62,6 +62,17 @@ wm_start()
     . /etc/default/locale
     export LANG LANGUAGE
   fi
+  
+  # DESKTOP_SESSION should be set in sesman.ini in the SessionVariables section.
+  # If set and valid then the STARTUP command will be taken from there
```

Comment:
Add a note that not all distros support this.

I can't find in in the CentOS 8 startup scripts for example.

Reply:
true.. What is a bit strange to me is that on debian does a exec in Xsession so post_start will never be called.

do you know if thats also the case in other distros?

if all do a exec then it might be better to call the Xsession script with the fisrt parameter being $STARTUP

Reply:
I don't think there's any real commonality between the distros. They all seem to do their own thing.

Another thing you could do is wrap this up in a function and call it from the debian start stanza. We can then add it to any others that will benefit from it.

Reply:
ok, i did a rewrite. now the function gives the startup command without setting the environment just by calling the function.

This way a user could also set the preferred desktop by putting DESKTOP_SESSION in .profile. In my previous version that did not work